tap_gitlab/streams.py

```diff
@@ -17,13 +30,13 @@ class ProjectsStream(ProjectBasedStream):
     replication_key = "last_activity_at"
     is_sorted = True
     extra_url_params = {"statistics": 1}
+    schema_filepath = None  # to allow the use of schema below
 
     def post_process(self, row: dict, context: Optional[dict] = None) -> Optional[dict]:
         result = super().post_process(row, context)
         if result is None:
             return None
 
-        result["owner_id"] = pop_nested_id(result, "owner")
```

Comment:
Shouldn't the "owner" appear in the schema now?

Reply:
Resolved in [67df00f](https://github.com/MeltanoLabs/tap-gitlab/pull/68/commits/67df00fc6c34e18c3e72c6c98e0b7896238988fd)